src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -1307,3 +1295,49 @@ def validate_match_condition(namespace):
     if namespace.if_none_match:
         namespace = _if_none_match(if_none_match=namespace.if_none_match, **namespace)
         del namespace.if_none_match
+
+
+def validate_or_policy(namespace):
+    error_elements = []
+    if namespace.properties is None:
+        error_msg = "Please provide --policy in JSON format or the following arguments: "
+        if namespace.source_account is None:
+            error_elements.append("--source-account")
+
+        # Apply account name when there is no destination account provided
+        if namespace.destination_account is None:
+            namespace.destination_account = namespace.account_name
+
+        if error_elements:
+            error_msg += ", ".join(error_elements)
+            error_msg += " to initialize or Policy for storage account."
+            raise ValueError(error_msg)
+    else:
+        if os.path.exists(namespace.properties):
+            or_policy = get_file_json(namespace.properties)
+        else:
+            or_policy = shell_safe_json_parse(namespace.properties)
+
+        try:
+            namespace.source_account = or_policy["sourceAccount"]
+        except KeyError:
+            namespace.source_account = or_policy["source_account"]
```

Comment:
Both sourceAccount and source_account are supported?

Reply:
yes